tests/console/check_package_version.pm

```diff
@@ -55,6 +60,14 @@ sub cmp_packages {
     }
 }
 
+sub loc_packages {
+    my ($pac, $loc) = @_;
+    record_info($pac, "$pac check package location");
+    my $output = script_output("zypper se -xs $pac | grep -w $pac | head -1 | awk -F '|' '{print \$6}'", 100, proceed_on_failure => 1);
+    diag "location:" . $output;
+    record_soft_failure("$loc doesn't have package $pac") if ($output !~ /$loc/);
```

Comment:
this is soft failure?

Reply:
Just follow version check: record_soft_failure("The $pcks is not existed");
We can record information first in case the feature not implement.

Reply:
i suggest using record_info intead, and this function should return error? then finally test case will set fail when finish check.

Reply:
Do we have record waring this kind of function can be used?

Reply:
Yes, at this file, we all use  record_soft_failure. I think it is OK at here. Don't impact others. Just record the failure reason.